native/src/components/MapView.tsx

```diff
@@ -47,7 +47,7 @@ type MapViewProps = {
 const featureLayerId = 'point'
 
 // Has to be set even if we use map libre
-MapboxGL.setAccessToken(mapConfig.accessToken)
+MapLibreGL.setAccessToken(null)
```

Comment:
please remove the `accessToken` if not needed anymore

Reply:
Still needed for now.

Reply:
where? couldn't find any usage of `mapConfig.accessToken`

Reply:
Ah. I thought you referred to `MapLibreGL.setAccessToken(null)`. I removed `mapConfig.accessToken`. Anything else to do here?